libs/expo/betterangels/src/lib/screens/AddEditClient/HMIS.tsx

```diff
@@ -1,25 +1,32 @@
 import { Spacings } from '@monorepo/expo/shared/static';
 import {
-  BasicInput,
   FieldCard,
+  Input,
   TextMedium,
 } from '@monorepo/expo/shared/ui-components';
 import { RefObject } from 'react';
+import { useFormContext } from 'react-hook-form';
 import { ScrollView, View } from 'react-native';
-import { CreateClientProfileInput } from '../../apollo';
+import {
+  CreateClientProfileInput,
+  UpdateClientProfileInput,
+} from '../../apollo';
 
 interface IHMISProps {
-  client: CreateClientProfileInput;
-  setClient: (client: CreateClientProfileInput) => void;
   expanded: undefined | string | null;
   setExpanded: (expanded: undefined | string | null) => void;
   scrollRef: RefObject<ScrollView>;
 }
 
 export default function HMIS(props: IHMISProps) {
-  const { expanded, setExpanded, client, setClient, scrollRef } = props;
+  const { expanded, setExpanded, scrollRef } = props;
+  const { control, watch } = useFormContext<
+    UpdateClientProfileInput | CreateClientProfileInput
+  >();
 
   const isHMIS = expanded === 'HMIS ID#';
```

Comment:
`isHMIS` -> `isHmis`?

Reply:
thought it is shorten word that is why left uppercase

Reply:
done

Reply:
it is, but I think acronyms usually get camel-cased the same as ordinary words https://stackoverflow.com/questions/15526107/acronyms-in-camelcase